test/FirebaseFunctionAndGoogleAssistant/functions/index.js

```diff
@@ -0,0 +1,11 @@
+const {dialogflow, BasicCard} = require("actions-on-google");
+const functions = require("firebase-functions")
+
+const app = dialogflow( {debug: true} );
```

Comment:
should  `debug `  be always `true`?

Reply:
good catch, it just outputs logs, but that is not necessary on the tests.